src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -419,4 +419,56 @@ public void addPoints_firstPoints(){
     
     Assert.assertEquals(expected, userEntity.getProperty("score"));
   }
+
+  // addUploadPoints when the user hasn't uploaded yet
+  @Test
+  public void addUploadPoints_firstPoints(){
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity user1 = new Entity("user");
+    user1.setProperty("userId", "newUser1");
+    datastore.put(user1);
+ 
+    UserUtils.addUploadPoints(user1, datastore);
+ 
+    Entity userEntity = UserUtils.getEntityFromDatastore("user", "userId", "newUser1", datastore);
+    long expected = 20;
+    
+    Assert.assertEquals(expected, userEntity.getProperty("score"));
```

Comment:
You could probably combine these lines and say Assert.assertEquals(20, blah);

Reply:
I get an error if I do that, it needs to be casted to long

Reply:
Alright, that's done -- with the add points function as well